dimod/quadratic/cyqm/cyqm_template.pyx.pxi

```diff
@@ -359,6 +359,10 @@ cdef class cyQM_template(cyQMBase):
             raise TypeError(f"cannot change vartype {self.vartype(v).name!r} "
                             f"to {vartype.name!r}") from None
 
+    def clear(self):
+        self.cppqm.clear()
+        self.variables._clear()
```

Comment:
`self.cppqm.clear()` calls back to `QuadraticModel::clear()` in `quadratic_model.hpp`, while `self.variables._clear()` calls `cyvariables.pyx`, right? Just trying to understand how everything fits together.

Reply:
Yes. For the models, Python calls Cython calls C++ :smile: 
For Variables you're correct as well.